__tests__/storage/common_store.test.js

```diff
@@ -177,7 +177,8 @@ describe('registered tokens', () => {
 
   it('should work with leveldb store', async () => {
     const xpriv = new HDPrivateKey();
-    const store = new LevelDBStore(DATA_DIR, xpriv.xpubkey);
+    const walletId = crypto.Hash.sha256(Buffer.from(xpriv.xpubkey)).toString('hex');
```

Comment:
Can we wait the other PR and use the util created here?

Reply:
If we do, I think it's important to also refactor the other instances of similar code in the tests ( about 14, on my latest search )